plugins/tools/plugin-cache/src/node/utils.ts

```diff
@@ -0,0 +1,47 @@
+import { createHash } from 'node:crypto'
+import { fs, path } from 'vuepress/utils'
+
+export const hash = (data: string): string =>
```

Comment:
Is it possible to make use of the `hash` util from `vuepress/utils`?

Reply:
I just noticed that `vuepress/utils` has `hash`, as long as the hash it generates is unique and stable, it is completely acceptable to use it.